infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -205,6 +210,15 @@ func (b *TFBlueprintTest) getTFOutputsAsInputs(o map[string]interface{}) map[str
 	return n
 }
 
+// loadSetupSAKey activate a gcp encoded sa key
+func (b *TFBlueprintTest) loadSetupSAKey(saEnc string) {
+	credDec, err := b64.StdEncoding.DecodeString(saEnc)
+	if err != nil {
+		b.t.Fatalf("Unable to decode setup sa key: %v", err)
+	}
```

Comment:
We may need to refactor this based on my comment above if we decide to accept plain SA key for explicit.